src/main/java/net/minecraftforge/common/data/GlobalLootModifierProvider.java

```diff
@@ -103,7 +103,6 @@ public void run(HashCache cache) throws IOException
      *
      * @param modifier      The name of the modifier, which will be the file name.
      * @param serializer    The serializer of this modifier.
-     * @param conditions    The loot conditions before {@link LootModifier#doApply} is called.
```

Comment:
No `instance` replacement?

Reply:
I consider adding that out of scope of this PR, as this PR is chiefly concerned with removing errors which prevent the `javadoc` tool from working.